metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "nmaludy-erlang",
+  "name": "puppet-erlang",
   "version": "0.1.0",
-  "author": "nmaludy",
+  "author": "puppet",
   "summary": "Manages repos and package Erlang installation",
   "license": "Apache-2.0",
-  "source": "https://github.com/nmaludy/puppet-erlang",
-  "project_page": "https://github.com/nmaludy/puppet-erlang",
-  "issues_url": "https://github.com/nmaludy/puppet-erlang/issues",
+  "source": "https://github.com/voxpupuli/puppet-erlang",
+  "project_page": "https://github.com/voxpupuli/puppet-erlang",
+  "issues_url": "https://github.com/voxpupuli/puppet-erlang/issues",
   "dependencies": [
     {
       "name": "puppetlabs/apt",
```

Comment:
I _think_ deps like this that are OS specific are supposed to be soft deps. But it’s been a bit

Reply:
https://puppet.com/docs/puppet/latest/style_guide.html#dependencies

> A soft dependency is a dependency that is only required in a specific set of use cases

That said, I have seen other modules require OS specific package managers.

Reply:
I think the dependencies are good as they are.